tests/Phinx/Db/Adapter/MysqlAdapterUnitTest.php

```diff
@@ -788,12 +806,33 @@ public function testGetSqlTypeExists()
         $this->assertEquals(array('name' => 'varchar', 'limit' => 255),
                             $this->adapter->getSqlType(MysqlAdapter::PHINX_TYPE_STRING));
         $this->assertEquals(array('name' => 'char', 'limit' => 255),
-                            $this->adapter->getSqlType(MysqlAdapter::PHINX_TYPE_CHAR));
+                            $this->adapter->getSqlType(MysqlAdapter::PHINX_TYPE_CHAR, 255));
+
+        //text combinations
+        $this->assertEquals(array('name' => 'tinytext'),
+                            $this->adapter->getSqlType(MysqlAdapter::PHINX_TYPE_TEXT, 512));
```

Comment:
you are right

I have updated test with constants when is equal or greater than limit.